users/management/commands/export_research.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from directory.models import Researches, ParaclinicInputGroups, ParaclinicInputField
+import json
+from appconf.manager import SettingManager
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('research_pk', type=str)
+
+    def handle(self, *args, **kwargs):
+        research = kwargs["research_pk"]
+        research_pk = int(research)
+        obj_research = {}
+        r = Researches.objects.get(pk=research_pk)
+        obj_research['title'] = r.title
+        obj_research['code'] = r.code
+        obj_research['short_title'] = r.short_title
+        groups = ParaclinicInputGroups.objects.filter(research=r)
+        t_group = []
+        for i in groups:
+            fieds = ParaclinicInputField.objects.filter(group=i, hide=False)
+            t_field = []
+            order = str(i.order)
+            for f in fieds:
```

Comment:
Можно сразу итерировать по queryset, тут вынос в переменную ничего не даёт:
```suggestion
            for f in ParaclinicInputField.objects.filter(group=g, hide=False):
```